gocourse08/webinar/main.go

```diff
@@ -0,0 +1,50 @@
+/*
+написати генерік функції:
+створення посилання на простий тип PtrOf() (int → *int)
+
+	пошук у слайсі елементу FindIn()
+
+видалення зайвих елементів у слайсі з метою залишити лише унікальні Unique()
+
+	func PtrOf(v int) *int {
+	    return &v
+	}
+*/
+package main
+
+import "fmt"
+
+func PtrOf[T int | float64](v T) *T {
```

Comment:
Адресу можна взяти з будь-якого типу:

```suggestion
func PtrOf[T any](v T) *T {
```

Глянути імплементацію реального коду можна тут:
https://github.com/dapr/kit/blob/v0.13.2/ptr/of.go#L21